setup.py

```diff
@@ -49,61 +53,24 @@
 
 
 setup(
-    name='JPype1',
-    version='1.5.0_dev0',
-    description='A Python to Java bridge.',
-    long_description=open('README.rst').read(),
-    license='License :: OSI Approved :: Apache Software License',
-    author='Steve Menard',
-    author_email='devilwolf@users.sourceforge.net',
-    maintainer='Luis Nell',
-    maintainer_email='cooperate@originell.org',
-    python_requires=">=3.7",
-    url='https://github.com/jpype-project/jpype',
+    # Non-standard, and extension behaviour. See also:
+    # https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html#setuptools-specific-configuration
     platforms=[
         'Operating System :: Microsoft :: Windows',
         'Operating System :: POSIX',
         'Operating System :: Unix',
         'Operating System :: MacOS',
     ],
-    classifiers=[
-        'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.7',
-        'Programming Language :: Python :: 3.8',
-        'Programming Language :: Python :: 3.9',
-        'Programming Language :: Python :: 3.10',
-        'Topic :: Software Development',
-        'Topic :: Scientific/Engineering',
-    ],
     packages=['jpype', 'jpype._pyinstaller'],
     package_dir={'jpype': 'jpype', },
     package_data={'jpype': ['*.pyi']},
-    install_requires=['typing_extensions ; python_version< "3.8"',
-        'packaging'],
-    tests_require=['pytest'],
-    extras_require={
-        'tests': [
-            'pytest',
-        ],
-        'docs': [
-            'readthedocs-sphinx-ext',
-            'sphinx',
-            'sphinx-rtd-theme',
-        ],
-    },
     cmdclass={
         'build_ext': setupext.build_ext.BuildExtCommand,
+        'develop': setupext.develop.Develop,
         'test_java': setupext.test_java.TestJavaCommand,
         'sdist': setupext.sdist.BuildSourceDistribution,
         'test': setupext.pytester.PyTest,
     },
     zip_safe=False,
     ext_modules=[jpypeJar, jpypeLib, ],
-    distclass=setupext.dist.Distribution,
-    entry_points={
```

Comment:
do we perform CI tests (invocations) for pyinstaller?

Reply:
Yes, I think so. It was introduced in https://github.com/jpype-project/jpype/pull/877/files#diff-42991dc440ef8e689b04809d09da836f085f961f771c3f929c1792b659bdb416R34, and the test runs at https://github.com/jpype-project/jpype/blob/master/.azure/scripts/test.yml#L41